src/main/resources/javascript/jcr-auth-provider/jcr-mapper-controller.js

```diff
@@ -55,6 +55,12 @@
                 return false;
             }
 
+            angular.forEach(vm.mapping, (mapping) => {
+                if (mapping.editable) {
+                    mapping.customMapper = true;
+                }
+                delete mapping.editable;
+            });
```

Comment:
Will it not be easier to leave this mapping editable the all time ? This would allow for renaming it and keep the input in place ?